ocfweb/docs/docs/staff/backend/libvirt.md

```diff
@@ -0,0 +1,80 @@
+[[!meta title="KVM/Libvirt"]]
+
+Most of the OCF's hosts are virtual machines. Our virtual machines are all
+powered by [QEMU][qemu]/[KVM][kvm] and managed by [libvirt][libvirt] on the
+hypervisors.
+
+VM disks are stored as LVM volumes on the hypervisors, typically under
+`/dev/vg/<vm-name>`.
+
+[qemu]: https://www.qemu.org/
+[kvm]: https://www.linux-kvm.org/
+[libvirt]: https://libvirt.org/
+
+
+## Administration
+
+### How do I view which VMs are on a hypervisor?
+
+SSH into the hypervisor, then run `sudo virsh list --all`:
+
+    kpengboy@hal:~$ sudo virsh list --all
+     Id    Name                           State
+    ----------------------------------------------------
+     1     fallingrocks                   running
+     3     limniceruption                 running
+     4     spriggs                        running
+     5     dev-flood                      running
+     11    dev-whiteout                   running
+     15    dev-death                      running
+     18    dev-tsunami                    running
+     -     dev-anthrax                    shut off
+     -     dev-pestilence                 shut off
+     -     dev-werewolves                 shut off
+     -     zombies                        shut off
+
+### How do I turn on a VM?
+
+On the hypervisor, run `sudo virsh start <vm-name>`.
```

Comment:
I think adding something in here about setting some VMs as autostart would be useful, because certain VMs are set to do that, but most are not and that could be a little confusing. Maybe mention that if it's a critical service, then it should be set to autostart with `sudo virsh autostart <vm-name>`?

Reply:
Done